### PR TITLE
Add Friendly Neighborhood Beeper plugin

### PR DESCRIPTION
**Pull request by techfreek**
Registers plugin that adds a completion beep during the specified hours

### CODE REVIEW

File: _plugins/FriendlyNeighborhoodBeeper.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- print finished
+- simple
+- control
+
+screenshots:
+- url: /assets/img/plugins/FriendlyNeighborhoodBeeper/settings.png
+  alt: Friendly Neighborhood Beeper settigns
+  caption: Config options for FNB. Current time is displayed as the OctoPrint host may be out of sync.
+
+featuredimage: /assets/img/plugins/FriendlyNeighborhoodBeeper/settings.png
```
Comment (jneilliii) [inline _plugins/FriendlyNeighborhoodBeeper.md:30]: This folder where you save your file and reference it from in your md file needs to match the id like this.

`featuredimage: /assets/img/plugins/FriendlyBeeper/settings.png`

File: _plugins/FriendlyNeighborhoodBeeper.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+tags:
+- notification
+- beep
+- print finished
+- simple
+- control
+
+screenshots:
+- url: /assets/img/plugins/FriendlyNeighborhoodBeeper/settings.png
```
Comment (jneilliii) [inline _plugins/FriendlyNeighborhoodBeeper.md:26]: This folder where you save your file and reference it from in your md file needs to match the id like this.

`featuredimage: /assets/img/plugins/FriendlyBeeper/settings.png`

File: _plugins/FriendlyNeighborhoodBeeper.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+follow_dependency_links: false
+
+tags:
+- notification
+- beep
+- print finished
+- simple
+- control
+
```
Comment (jneilliii) [inline _plugins/FriendlyNeighborhoodBeeper.md:24]: Missing Python 3 compatibility flag. Looks like your plugin has it but the md file here doesn't.
```
compatibility:
  python: ">=2.7,<4"
```

File: _plugins/FriendlyNeighborhoodBeeper.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+layout: plugin
+
+id: FriendlyBeeper
+title: OctoPrint-Friendlybeeper
```
Comment (jneilliii) [inline _plugins/FriendlyNeighborhoodBeeper.md:5]: Not required, but you may want to change this to match your plugin name for consistency.

`title: Friendly Neighborhood Beeper`

File: _plugins/FriendlyNeighborhoodBeeper.md (hunk 3 above)
Comment (techfreek) [reply _plugins/FriendlyNeighborhoodBeeper.md:24]: Sorry about that, I got confused by this comment in the tutorial:
```
# You only need the following if your plugin requires specific OctoPrint versions or
# specific operating systems to function - you can safely remove the whole
# "compatibility" block if this is not the case.
```
I'll get this fixed

**Comment by techfreek** [comment]
Feedback should be addressed. Let me know if anything else needs to be fixed

**Comment by jneilliii** [comment]
Thanks @techfreek looks good, merged.

**Pull request merged**